packages/admin-panel-server/src/viz-builder/dashboardVisualisation/DashboardVisualisationCombiner.ts

```diff
@@ -34,7 +34,7 @@ export class DashboardVisualisationCombiner {
     const { config: reportConfig } = this.report;
     const { output } = reportConfig;
     const { config: dashboardItemConfig } = this.dashboardItem;
-    const { type, ...config } = dashboardItemConfig;
+    const { type, name, ...config } = dashboardItemConfig;
```

Comment:
The `name` of the viz should be updated by the Main Edit modal of the viz-builder, not through chart config. So this is to excluding it from the chart config